sqla_vertica_python/vertica_python.py

```diff
@@ -135,6 +135,11 @@ def get_schema_names(self, connection, **kw):
         return [row[0] for row in rs if not row[0].startswith('v_')]
 
 
+    @reflection.cache
+    def get_table_comment(self, connection, table_name, schema=None, **kw):
+        raise NotImplementedError
```

Comment:
@jeffreythewang from my understanding, Vertica supports table comments. Do you have access to a Vertica database? I believe that implementing this method should be relatively easy if you have a database to test against.